quinn-h3/src/qpack/encoder.rs

```diff
@@ -214,14 +214,23 @@ impl From<ParseError> for Error {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::qpack::dynamic::SETTINGS_HEADER_TABLE_SIZE_DEFAULT as TABLE_SIZE;
+    const TABLE_SIZE: usize = 4096;
+
+    fn build_table() -> DynamicTable {
```

Comment:
And yet another one! Maybe it would make sense to have a module that has a single copy of this that all the tests can use?